framework/src/Degradations/PhantomCharacter.cpp

```diff
@@ -892,7 +887,7 @@ degradeComposant(cv::Mat &output,
 }
 
 cv::Mat
-phantomCharacter(const cv::Mat &imgOriginal, Frequency frequency)
+phantomCharacter(const cv::Mat &imgOriginal, Frequency frequency, QString phantomPatternsPath)
```

Comment:
You should pass a const reference here:
```suggestion
phantomCharacter(const cv::Mat &imgOriginal, Frequency frequency, const QString &phantomPatternsPath)
```